packages/material-ui/src/Chip/Chip.js

```diff
@@ -421,7 +426,8 @@ const Chip = React.forwardRef(function Chip(props, ref) {
         themeVariantsClasses,
         className,
       )}
-      aria-disabled={disabled ? true : undefined}
+      aria-disabled={!clickable && disabled ? true : undefined}
+      disabled={clickable && disabled ? true : undefined}
```

Comment:
shouldn't it be `!clickable` here as well? The same way we set `aria-disabled`?
```suggestion
      disabled={!clickable && disabled ? true : undefined}
```

Reply:
I don't think so, the aria-disabled is meant for when we don't use a ButtonBase.